h2/src/main/org/h2/mvstore/cache/CacheLongKeyLIRS.java

```diff
@@ -246,7 +247,7 @@ private int getSegmentIndex(int hash) {
      * @param key the key
      * @return the hash code
      */
-    static int getHash(long key) {
+    private static int getHash(long key) {
```

Comment:
This method should have default visibility as it was before to avoid synthetic access.